src/command.cpp

```diff
@@ -81,16 +84,24 @@ bool Command::checkError(Error& errorDetails, JSON& json)
                        errorDetails.setLinkStatus(json.getint());
                         break;
                     case EOO:
-                        return true;
+                        exit = true;
```

Comment:
this now can entail returning false (if !errorDetected). could that be a problem?

Reply:
I don't think so. It's made in purpose: if no errocode is read from the JSON (whether `[<error_code>]` or `{"err":<error_code>, ...}`), then there is no error detected at this stage.